src/HealthChecks.UI.InMemory.Storage/HealthChecksUIBuilderExtensions.cs

```diff
@@ -6,12 +6,12 @@ namespace Microsoft.Extensions.DependencyInjection
 {
     public static class HealthChecksUIBuilderExtensions
     {
-        public static HealthChecksUIBuilder AddInMemoryStorage(this HealthChecksUIBuilder builder, Action<DbContextOptionsBuilder> configureOptions = null)
+        public static HealthChecksUIBuilder AddInMemoryStorage(this HealthChecksUIBuilder builder, Action<DbContextOptionsBuilder> configureOptions = null, string databaseName = "HealthChecksUI")
         {
             builder.Services.AddDbContext<HealthChecksDb>(options =>
             {
                 configureOptions?.Invoke(options);
-                options.UseInMemoryDatabase("HealthChecksUI");
+                options.UseInMemoryDatabase(databaseName);
```

Comment:
Exposed new default parameter databaseName for inmemory provider. We have some functional tests that check the migrations contents and they can share context.